lib/mix/tasks/phoenix.gen.html.ex

```diff
@@ -87,6 +87,17 @@ defmodule Mix.Tasks.Phoenix.Gen.Html do
     """
   end
 
+  defp labels(attrs) do
```

Comment:
Since this is duplicate from the inputs, i think we should change how inputs are generated. maybe make them tuples with three elements, containing the form function, the input name, and then the rest of the information.